.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
```

Comment:
The CI [stopped working on 6](https://travis-ci.com/github/bugsnag/bugsnag-sourcemaps/jobs/310546408#L208) but I couldn't reproduce locally.

Took the decision to update the CI matrix rather than figure out testing on a [deprecated](https://nodejs.org/en/about/releases/) version of Node.